src/components/Reports/MPGAIncomeExpensesReport/Charts/SummaryBarChart.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
```

Comment:
[Must-fix] Missing required imports for proper internationalization. Evidence: No imports for `useLocale` and `currencyFormat`. Impact: Cannot implement proper locale-aware formatting. Fix: Add missing imports.

```diff
+ import { useLocale } from 'src/hooks/useLocale';
+ import { currencyFormat } from 'src/lib/intlFormat';
```